pod/perlguts.pod

```diff
@@ -1023,6 +1023,62 @@ as any other SV.
 
 For more information on references and blessings, consult L<perlref>.
 
+=head2 I/O Handles
+
+Like AVs and HVs, IO objects are another type of non-scalar SV which
+may contain input and output L<PerlIO|perlapio> objects or a C<DIR *>
+from opendir().
+
+You can create a new IO object:
+
+    IO*  newIO();
+
+The IO itself has little distinct function from other SVs, it's just a
```

Comment:
I found myself re-reading this line trying to understand it.

Reply:
I've removed it.  I was trying to make the point that there's no (or at least no documented) io_* functions like there are sv_*, av_* and hv_* functions. (io_close() exists but isn't API), but it isn't really necessary